package.json

```diff
@@ -12,6 +12,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.21.1",
```

Comment:
Isn't it already there? Probably need to rebase on master

Reply:
I will create new branch from actual master and then cherry-pick commits with tests from this branch.